src/sections/image_resources_section.rs

```diff
@@ -0,0 +1,761 @@
+/// | 4                                                     | Type: OSType key                                                                           |
+/// |                                                       | 'obj ' = Reference                                                                         |
+/// |                                                       | 'Objc' = Descriptor                                                                        |
+/// |                                                       | 'VlLs' = List                                                                              |
+/// |                                                       | 'doub' = Double                                                                            |
+/// |                                                       | 'UntF' = Unit float                                                                        |
+/// |                                                       | 'TEXT' = String                                                                            |
+/// |                                                       | 'enum' = Enumerated                                                                        |
+/// |                                                       | 'long' = Integer                                                                           |
+/// |                                                       | 'comp' = Large Integer                                                                     |
+/// |                                                       | 'bool' = Boolean                                                                           |
+/// |                                                       | 'GlbO' = GlobalObject same as Descriptor                                                   |
+/// |                                                       | 'type' = Class                                                                             |
+/// |                                                       | 'GlbC' = Class                                                                             |
+/// |                                                       | 'alis' = Alias                                                                             |
+/// |                                                       | 'tdta' = Raw Data                                                                          |
+/// | Variable                                              | Item type: see the tables below for each possible type                                     |
+/// +-------------------------------------------------------+--------------------------------------------------------------------------------------------+
+#[derive(Debug)]
+pub struct DescriptorStructure {
+    pub name: String,
+    pub fields: HashMap<String, DescriptorField>,
```

Comment:
Should this string be something like

```rust
#[derive(Debug, Hash, Eq, PartialEq)]
enum DescriptorKey {
    Reference,
    Descriptor,
    // ...
    String(String)
}
```

Since it looks like this can either be a specific OSType key otherwise a String?

Reply:
This string is not `OSType`, it is just item name
![image](https://user-images.githubusercontent.com/15220500/78323697-4ce93300-757b-11ea-82f1-01a074347f87.png)

Reply:
Gotcha